src/post_setup.sh

```diff
@@ -1,21 +1,9 @@
 #!/bin/env bash
 
-set -o nounset
-set -o errexit
-set -o pipefail
-
-# Moved out of packer.pkr.hcl's inline provisioner shell to here.
-# These commands come from here:
-# https://github.com/cisagov/debian-packer/blob/e34570f730a04b680bcff5313384a1a13b4c935a/src/packer.pkr.hcl#L129
-sed --in-place '/^users:/ {N; s/users:.*/users: []/g}' /etc/cloud/cloud.cfg
-rm --force /etc/sudoers.d/90-cloud-init-users
-rm --force /root/.ssh/authorized_keys
-/usr/sbin/userdel --remove --force ubuntu
-
```

Comment:
Why was all of this removed?  Was this unintentionally lost during the merge from 'develop'?

Reply:
It was put back into `packer.pkr.hcl` because it was failing when run from the `post_setup.sh`. 

[89b57fa](https://github.com/cisagov/ubuntu-server-packer/pull/15/commits/89b57fa7fbf2f502e05832476cabd6f967443f9b)and
[7e0d0e5](https://github.com/cisagov/ubuntu-server-packer/pull/15/commits/7e0d0e5cf2731e1ec358782aed11d4ccd3075a35)

Reply:
Ah, my bad - I missed that those commands were relocated back to where they were originally.

But it still doesn't explain why you removed our standard shell script options:
```
set -o nounset
set -o errexit
set -o pipefail
```

Reply:
That was unintentional. I put them back.

See [4ce648d](https://github.com/cisagov/ubuntu-server-packer/commit/4ce648d9d37fbd34f9f4f0f5a4693f7fd2c35613)